pkg/globalnet/controllers/pod_controller.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
How about we name the file _ingress_pod_controller_ to distinguish? Later on we can name the the other one _egress_pod_controller_.